scipy-1.0/paper.tex

```diff
@@ -592,7 +592,18 @@ \subsection*{Roadmap}
 
 \subsection*{Community beyond the SciPy library}
 
-\fixme{TODO}
+The scientific Python ecosystem includes many examples
+of domain-specific software libraries building on top
+of SciPy features, and then returning to the base SciPy library
+to suggest and even implement improvements to enable
+more effective science applications. For example, there
+are common contributors to the SciPy and astropy core
+libraries\cite{astropy-2018}, and what works well for 
+one of the code bases, infrastructures, or communities 
+is often transferred in some form to the other. At the code
+base level, the \texttt{binned\_statistics} functionality
```

Comment:
what do you think about: 

At the code base level, the \texttt{binned\_statistics} functionality _is one of such cross-project contributions: it_ was ...

just to clarify that this is an "example of the example"